autobahn/websocket/protocol.py

```diff
@@ -2758,7 +2759,10 @@ def succeedHandshake(self, res):
         if self.factory.server is not None and self.factory.server != "":
             response += "Server: %s\x0d\x0a" % self.factory.server
 
-        response += "X-Powered-By: AutobahnPython/{0}\x0d\x0a".format(__version__)
+        if self.showServerVersion:
+            response += "X-Powered-By: AutobahnPython/{0}\x0d\x0a".format(__version__)
+        else:
+            response += "X-Powered-By: AutobahnPython\x0d\x0a".format(__version__)
```

Comment:
Should probably not have a __version__ formatting arg - but on the other hand, maybe do away with "X-Powered-By" header altogether. One server header is enough.